### PR TITLE
--securitygroup flag for disco_aws.py

### PR DESCRIPTION
**Pull request by IlyaSukhanov**
Pushed to master rather than branch. So opening a PR to view change in reverse pattern.. so we can get it code reviewed.

### CODE REVIEW

**Comment by pchand-amplify** [comment]
Can you elaborate on why this is required?

File: bin/disco_aws.py [hunk 1]
```diff
@@ -293,6 +296,8 @@ def run():
             if args.productline or args.all:
                 productline = instance.tags.get("productline", u"unknown")
                 line += u" {0:<15}".format(productline if productline != u"unknown" else u"-")
+            if args.securitygroup or args.all:
+                line += u" {0:15}".format(instance.groups[0].name)
```
Comment (jichen-amplify) [inline bin/disco_aws.py]: Do you always want the first group if there are more than one item in `instance.groups`?

Comment (IlyaSukhanov) [inline bin/disco_aws.py]: We don't support for multiple SGs.

**Comment by IlyaSukhanov** [comment]
This is not required. Its a creature comfort. It aids in troubleshooting problems. You want to correlate problematic instances to other configs. This gives you some insight on sg and metanetwork that the instance is in. Previously you could infer this from IPs (although even this was bothersome), today with dynamic IP allocation to metanetworks its not easy.

**Comment by jichen-amplify** [comment]
👍

File: bin/disco_aws.py (hunk 1 above)
Comment (kelibezhani) [inline bin/disco_aws.py]: If you know the *metanetwork* of an instance, you have the security group.

**Comment by IlyaSukhanov** [comment]
Since this is already on master I'm just closing this PR.

**Pull request closed without merging**